src/crimson/os/seastore/extentmap_manager.h

```diff
@@ -0,0 +1,119 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=8 sw=2 smarttab
+
+#pragma once
+
+#include <iostream>
+
+#include <boost/intrusive_ptr.hpp>
+#include <boost/smart_ptr/intrusive_ref_counter.hpp>
+
+#include <seastar/core/future.hh>
+
+#include "common/Formatter.h"
+#include "include/ceph_assert.h"
+#include "include/buffer_fwd.h"
+
+#include "crimson/osd/exceptions.h"
+#include "crimson/os/seastore/seastore_types.h"
+#include "crimson/os/seastore/transaction_manager.h"
+
+#define PAGE_SIZE 4096
+#define EXTMAP_BLOCK_SIZE 4096
+
+namespace crimson::os::seastore {
+
+struct lext_map_val_t {
+  laddr_t laddr;
+  extent_len_t length = 0;
+
+  lext_map_val_t(
+    laddr_t laddr,
+    extent_len_t length)
+    : laddr(laddr), length(length) {}
+
+};
+class Extent
```

Comment:
I don't really want to call this Extent for a few reasons:
- as this is really just a simple collection of ints, it's more of a simple struct, so it should be lower case extent_t
- other things are called extents already

Call it extent_mapping_t or something.